aya/src/util.rs

```diff
@@ -143,6 +144,25 @@ macro_rules! include_bytes_aligned {
     }};
 }
 
+pub(crate) fn page_size() -> usize {
+    // Safety: libc
+    (unsafe { sysconf(_SC_PAGESIZE) }) as usize
+}
+
+// while not currently used, it feels like this may be useful
+// for some of the alignment FIXMEs so leaving in place for now
+#[allow(dead_code)]
+// round_up is used to round a value X up to the nearest Y
+pub(crate) fn round_up(x: usize, y: usize) -> usize {
```

Comment:
I don't think we need this. The alignment comments are about something else - pointers needing to be aligned before rust references can be created from them, which is not something that this can help with. Also, I'm pretty sure there's already two other versions of this, one in the BTF code and one in the netlink code

Reply:
ack, it was handy to page align offsets but i've removed it for now.